code/modules/security levels/keycard authentication.dm

```diff
@@ -104,6 +109,10 @@ var/global/list/obj/machinery/keycard_auth/authenticators = list()
 		to_chat(usr, "This device is without power.")
 		return
 	if(href_list["triggerevent"])
+		if(href_list["triggerevent"] == "Emergency Response Team")
+			ert_reason = stripped_input(usr, "Please input the reason for calling an Emergency Response Team. This may be all the briefing they get before arriving at the station.", "Response Team Justification", ert_reason)
+			if(!ert_reason || !Adjacent(usr) || usr.incapacitated())
+				return
```

Comment:
You should do a to_chat to the user if there's no reason given instead of a silent failure.